packages/runtime/src/core/index.ts

```diff
@@ -16,8 +17,12 @@ export {
   ControlInstance,
   DefaultControlInstance,
   ShapeV2Control,
+  getBaseBreakpoint,
+  getBreakpoint,
   findBreakpointOverride,
+  findNextFallback,
   mergeOrCoalesceFallbacks,
   mergeResponsiveValues,
   shallowMergeFallbacks,
+  getViewportStyle,
```

Comment:
Re-exporting the rest of the breakpoint ops in under the `@makeswift/runtime/core` entrypoint for the builder.